frontend/scss/contexture.scss

```diff
@@ -0,0 +1,42 @@
+﻿@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;700&display=swap');
+@import "bootstrap-colors";
+@import "bootstrap-required";
+@import "bootstrap-variables";
+@import "bootstrap";
+
+h2 {
+  text-transform: uppercase;
+}
+
+h3 {
+  color: $primary;
+}
+
+.text-danger, .text-success, .text-info {
+  font-size: 0.875rem;
+}
+
+.text-danger, .text-success {
+  font-weight: 600;
+}
+
+// Colors
+
+section {
```

Comment:
I think that using a semantic css class in contexture is better suited than changing the default behavior of section

Reply:
I will move these styles since they're purely for the UI elements page